tests/environment.test.js

```diff
@@ -97,5 +97,41 @@ tap.test('Test Environment variables', test => {
     assert.end()
   })
 
+  test.test('Should fail since CONDITION_FIELD is true and CONDITION_TRUE_REQUIRED_FIELD is not present', async assert => {
```

Comment:
Is it possible to add a positive test (one that verifies whether_if-then-else_ configuration allows to start the service successfully) and test with `oneOf` and `anyOf`?  
Adding the latter tests would cover the whole use case described in #231.

Reply:
Done

Reply:
Ok, sorted :)